docs/CONTRIBUTING.md

```diff
@@ -55,13 +55,13 @@ Acts as integration branch for feature and bugfix branches that should go into t
 
 ### Feature branches
 
-Created from develop and should be named feature/<issue id in github>-<short description>. For example to work on "User Story 35: Remove the 'Classic' link stage in the API and only use permanent links" you would create a branch from develop named "feature/35-remove-classic-links". Note that the <short description> is all lower-case with hyphens.
+Created from develop and should be named feature/<issue id in github>-<short description>. For example to work on "User Story 35: Remove the 'Classic' link stage in the API and only use permanent links" you would create a branch from develop named "feature/35-remove-classic-links". Note that the `<short description>` is all lower-case with hyphens.
```

Comment:
You have missed this one: `feature/<issue id in github>-<short description>`

Reply:
Thanks! Fixed now